### ISSUE TITLE
Non Existent Key throws an Exception , Is this intended ?

### ISSUE THREAD

**Issue by Tasmanakos** [aksoftware98/multilanguages#49]
In Class Keys , Method GetValue() (Line 153)

```cs
object obj = keyValues[key];
if (obj == null)
{
    return key;
}
```

if the "key" does not exist it throws a KeyNotFound Exception . 
Is this intended ? because in Ver8 it was actually returning the  "key" value to String().
Thanks !
Edited : Great Library by the way has helped me a lot ;-)

**Comment by aksoftware98** [comment]
Thanks for letting me know, and thanks for that nice feedback. 

The fix is coming next week

**Comment by aksoftware98** [comment]
@Tasmanakos 

Sorry for being that late, but it has been a super busy year, waited for the holidays

Happy New Year and thanks for using the package.

The issue has been addressed in the new release 05.9.12.0

**Comment by Tasmanakos** [comment]
@aksoftware98 
Excellent Thank you very much for spending time to fix this ! 
Happy new Year , health above all else .
